portfolio/central/display_time.py

```
@@ -0,0 +1,41 @@
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+_DEFAULT_TZ = os.environ.get("OPENFDD_DISPLAY_TZ", "").strip()
+
+
+def display_tz() -> ZoneInfo:
+    if _DEFAULT_TZ:
+        try:
+            return ZoneInfo(_DEFAULT_TZ)
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)